app/assets/stylesheets/custom.css.erb

```diff
@@ -188,3 +188,29 @@ button.glyphicon {
   background-color: #9c5619;
   border-color:`` #9c5619;
 }*/
+
+/*vendor index styling*/
+
+.container {
+  border: 1px solid;
+  border-color: red;
+
+}
+
+.category {
+  border: 1px solid;
+  border-color: purple;
```

Comment:
I wanted to leave these as a guide for how they would look as a unit with the container, especially when we seed the images in.